Dockerfile

```diff
@@ -1,21 +1,16 @@
-FROM public.ecr.aws/docker/library/fedora:36 as base
+FROM public.ecr.aws/docker/library/fedora:37 as base
 
 # Everything we need to build our SDK and packages.
 RUN \
   dnf makecache && \
   dnf -y update && \
-  dnf -y groupinstall "C Development Tools and Libraries" && \
   dnf -y install --setopt=install_weak_deps=False \
-    rpmdevtools dnf-plugins-core createrepo_c \
-    cmake git meson perl-ExtUtils-MakeMaker python which \
-    bc hostname intltool gperf kmod rsync wget openssl \
-    dwarves elfutils-devel libcap-devel openssl-devel \
-    createrepo_c e2fsprogs gdisk python3-jinja2 \
-    kpartx lz4 veritysetup dosfstools mtools squashfs-tools \
-    perl-FindBin perl-IPC-Cmd perl-open policycoreutils \
-    secilc qemu-img glib2-devel rpcgen erofs-utils jq ShellCheck \
-    json-c-devel libcurl-devel p11-kit-devel && \
-  dnf clean all && \
+    bc bison cmake cpio curl dnf-plugins-core dwarves elfutils-devel flex g++ \
+    gcc git gperf hostname intltool jq json-c-devel kmod libtool libcurl-devel \
+    meson openssl openssl-devel p11-kit-devel perl-ExtUtils-MakeMaker \
+    perl-FindBin perl-IPC-Cmd perl-open python rsync wget which && \
```

Comment:
I would probably put one per line and sort it so that changes to the list are discernible. That's a debatable style choice of course...

Reply:
Good point - this is definitely the right time to do that, so everything is easier to review going forward.